bootstrap.sh

```diff
@@ -66,23 +66,42 @@ fi
 git submodule update --init --recursive
 
 kernel_ver=`uname -r`
+kernel_hed=`sudo apt search linux-headers-$(uname -r)`
+
 echo "Running kernel version: $kernel_ver"
+echo "..................................."
+echo "Running kernel headers: $kernel_hed"
```

Comment:
Please use sensible naming. Rename this to kernel_header_ver

Reply:
Updated the script with proper variable names, thanks.